scout/commands/download/ensembl.py

```diff
@@ -8,9 +8,20 @@
 
 from scout.utils.ensembl_biomart_clients import EnsemblBiomartHandler
 
+CHROM_SEPARATOR = "[success]"
+NR_EXPECTED_CHROMS = 24
+
 LOG = logging.getLogger(__name__)
 
 
+def integrity_check(nr_chromosomes_in_file: int):
+    if nr_chromosomes_in_file < NR_EXPECTED_CHROMS:
+        raise BufferError(
+            "Ensembl resource does not seem to be complete. Please retry downloading the file."
+        )
+    LOG.info("Integrity check OK.")
```

Comment:
```suggestion
    LOG.info("Ensembl download integrity check OK.")
```
or sth to identify what the integrity is for. Maybe not even needed? 😊

Reply:
It's referred to the file that is printed immediately before. I don't think it's needed